client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -330,24 +391,38 @@ class DomainDetailsForm extends Component {
 		);
 	}
 
+	renderExtraDetailsForm = () => {
+		return ( <ExtraInfoFrForm
+			isProbablyOrganization={ Boolean( formState.getFieldValue( this.state.form, 'organization' ) ) }
+			values={ this.state.registrantExtraInfo }
+			onStateChange={ this.handleExtraChange } >
+				{ this.renderSubmitButton() }
+			</ExtraInfoFrForm>
+		);
+	}
+
 	handleCheckboxChange = () => {
 		this.setPrivacyProtectionSubscriptions( ! this.allDomainRegistrationsHavePrivacy() );
 	}
 
-	closeDialog = () => {
-		this.setState( { isDialogVisible: false } );
+	closeDialog( dialogKey = 'isDialogVisible' ) {
```

Comment:
Good catch! They do need to be bound to work when passed in to the `PrivacyProtection` component [here](https://github.com/Automattic/wp-calypso/blob/68273016cb2e6581a807f23bb08788cdb3a65c4b/client/my-sites/upgrades/checkout/domain-details-form.jsx#L267)